testing/cli/test_cli_sct_analyze_lesion.py

```diff
@@ -116,9 +116,9 @@ def compute_expected_measurements(dim, starting_coord=None, path_seg=None, mid_s
     ([(29, 45, 25), (3, 10, 2)], 0.001),
     ([(29, 27, 25), (1, 4, 1)], 0.001),  # NB: Example from #3633
     # Curved region of `t2.nii.gz` -> lots of curvature -> larger tolerance
-    ([(29, 0, 25), (4, 15, 3)], 0.01),
+    ([(31, 0, 25), (4, 15, 3)], 0.01),
     # Multiple lesions
-    ([[(29, 0, 25), (4, 15, 3)],
+    ([[(31, 0, 25), (4, 15, 3)],
```

Comment:
I took the liberty to slightly shift the lesion to be entirely in the spinal cord.

<details><summary>old: (29, 0, 25)</summary>

<img width="500" alt="Screenshot 2024-12-13 at 3 39 34 PM" src="https://github.com/user-attachments/assets/5cda4f4d-6755-4f5b-b059-5da474c5286b" />

</details>

<details><summary>new (31, 0, 25)</summary>

<img width="500" alt="Screenshot 2024-12-13 at 3 40 51 PM" src="https://github.com/user-attachments/assets/7797129d-6794-48a1-9a57-c56d0d233e15" />

</details>